pkg/node/manager/manager.go

```diff
@@ -404,15 +512,34 @@ func (m *manager) backgroundSync(ctx context.Context) error {
 		case <-startWaiting:
 		}
 
-		hr := m.health.NewScope("background-sync")
 		if err != nil {
-			hr.Degraded("Failed to apply node validation", err)
+			health.Degraded("Failed to apply node validation", err)
 		} else {
-			hr.OK("Node validation successful")
+			health.OK("Node validation successful")
 		}
 	}
 }
 
+const (
+	upAndRunningFlags    = unix.IFF_UP | unix.IFF_RUNNING
+	slaveOrLoopbackFlags = unix.IFF_SLAVE | unix.IFF_LOOPBACK
+)
+
+func deviceIsUpAndRunning(dev *tables.Device) bool {
+	if dev.Type != "device" {
```

Comment:
Nit. Could be worth adding a comment about this check for future reference, given that the comparison is not against a descriptive constant.